backend/app/tasks/subscription_tasks.py

```
@@ -1209,13 +1209,40 @@ def execute_subscription_task(
+                        )
+                        loop.run_until_complete(
+                            subscription_notification_dispatcher.dispatch_execution_notifications(
+                                db,
+                                subscription_id=subscription_id,
+                                execution_id=execution_id,
+                                subscription_display_name=subscription_display_name,
+                                result_summary=result_summary,
+                                status=BackgroundExecutionStatus.COMPLETED.value,
+                            )
+                        )
+                    except Exception as notify_error:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)